output/list.go

```diff
@@ -32,17 +32,8 @@ var green = color.New(color.FgGreen).SprintFunc()
 var yellow = color.New(color.FgYellow).SprintFunc()
 var red = color.New(color.FgRed).SprintFunc()
 
-// Summary contains an account of the test run
-type Summary struct {
-	Failed bool
-
-	SuccessAmount int
-	FailAmount    int
-	SkippedAmount int
-}
-
-// Stream results to the console, error at end if any fail
-func Stream(tests <-chan execute.TestResponse) Summary {
+// List is a ReporterFunc that produces a verbose list of results
+func List(tests <-chan execute.TestResponse) Summary {
```

Comment:
FYI: List is not actually a ReporterFunc. You can optionally do the following to actually make it a `ReporterFunc` so that you don't have to cast it later.

```go
var List ReporterFunc = func(tests <-chan executer.TestResponse) Summary {
  // ...
}
```